bluemira/codes/_freecadconfig.py

```diff
@@ -91,21 +91,30 @@ def _patch_pivy():
     import pivy  # noqa: PLC0415
 
     if pivy.__version__ == "0.6.9":
-        subprocess.run(  # noqa: S602
-            [
-                "/usr/bin/patch",
-                "-l",
-                "-p2",
-                "-N",
-                "-d",
-                f"{Path(pivy.__file__).parent}",
-                "<",
-                "pivy.patch",
-            ],
-            cwd=Path(__file__).parent,
-            shell=True,
-            check=False,
-        )
+        try:
+            subprocess.run(  # noqa: S602
+                [
+                    "/usr/bin/patch",
+                    "-l",
+                    "-p2",
+                    "-N",
+                    "-d",
+                    f"{Path(pivy.__file__).parent}",
+                    "<",
+                    "pivy.patch",
+                ],
+                cwd=Path(__file__).parent,
+                shell=True,
+                check=False,
+                timeout=3,
+            )
+        except subprocess.TimeoutExpired:
+            raise TimeoutError(
+                "Patch failed to apply please run manually in the repo root:\n"
```

Comment:
We could make a bash script for it and say that here too.

"Run bash ./scripts/patch_picy.sh"

Kinda thing

Reply:
I think making a script is probably overkill for a 2 line temporary command

Reply:
made a single line